cmd/ocm-backplane/cloud/common.go

```diff
@@ -297,7 +298,22 @@ func (cfg *QueryConfig) getIsolatedCredentials(ocmToken string) (aws.Credentials
 		Credentials: NewStaticCredentialsProvider(seedCredentials.AccessKeyID, seedCredentials.SecretAccessKey, seedCredentials.SessionToken),
 	})
 
-	inlinePolicy, err := getTrustedIPInlinePolicy()
+	clientIP, err := checkEgressIP(http.DefaultClient, "https://checkip.amazonaws.com/")
```

Comment:
Is the `http.DefaultClient` configured with the proxy?

Reply:
I think it is based on my testing

Reply:
Good catch, Sam. Considering that we already have `proxy-url` configured in the backplane config, it probably makes more sense to use a custom `http.Client` with the proxy explicitly configured based on `cfg.BackplaneConfiguration.ProxyURL` for `checkEgressIP()` rather than rely on `http.DefaultCLient` which rely on env variables?

Reply:
```
http.DefaultTransport = &http.Transport{Proxy: http.ProxyURL(proxyURL)}
```
@bmeng , I guess you can specify the backplane config proxy, so it will make sure the local IP via configured proxy. 
https://github.com/openshift/backplane-cli/blob/main/pkg/utils/clientUtils.go#L62